.circleci/config.yml

```diff
@@ -188,17 +202,78 @@ jobs:
           paths:
             - .
 
+  android_test:
+    executor: android
+    steps:
+      - checkout
+
+      - attach_workspace:
+          at: ~/player
+
+      - restore_cache:
+          keys:
+            - v1-bazel-cache-core-{{ .Branch }}-{{ .Revision }}
+            - v1-bazel-cache-core-{{ .Branch }}
+            - v1-bazel-cache-core-main
+
+      - run:
+          name: Install Bazel
+          command: |
+            wget -q https://github.com/bazelbuild/bazelisk/releases/download/v1.11.0/bazelisk-linux-amd64 -O bazelisk && \
+            chmod +x bazelisk && \
+            sudo mkdir /opt/bazelisk-v1.11.0 && \
+            sudo mv bazelisk /opt/bazelisk-v1.11.0 && \
+            sudo ln -s /opt/bazelisk-v1.11.0/bazelisk /usr/local/bin/bazel
+
+      # This is done because the outputs from the prev build can't be used due to lack of flag
+      # .bazelrc doesn't currently support string interpolation
+      - run: bazel build --define=APPLITOOLS_KEY=${APPLITOOLS_KEY} //android/demo:android_instrumentation_test
```

Comment:
@sugarmanz I know you asked for this to be removed, I think I can potentially do something like `bazel test --define...` and it would still build it, but I don't think it's any less work than explicitly building. 

Doing that also would require me to move the install step into the script right before the test, and I'd have to go all the way to the root before finding a path to the APKs for installation because the workspace is different within the script execution (doable but slightly questionable?)

Reply:
> I think I can potentially do something like bazel test --define... and it would still build it, but I don't think it's any less work than explicitly building.

Not having `--define=APPLITOOLS_KEY=${APPLITOOLS_KEY}` on the `bazel test` command actually causes the test module to be rebuilt.

![Screen Shot 2022-07-21 at 6 24 47 PM](https://user-images.githubusercontent.com/9255651/180325380-3befd306-4d45-4928-b8f3-96de30717aa6.png)

So, it should be on both if the commands are done separately.

> Doing that also would require me to move the install step into the script right before the test, and I'd have to go all the way to the root before finding a path to the APKs for installation because the workspace is different within the script execution (doable but slightly questionable?)

I think this is what you were saying about needing the `$BUILD_WORKSPACE_DIRECTORY`. However, I think that's probably an incorrect usage. The script should be giving all the context including the APK paths (I think as part of runfiles?) w/ the `data` params to actually perform the installation dynamically from where it is called.